controllers/ArticleController.js

```diff
@@ -138,12 +182,11 @@ class ArticleController {
         success: true,
         article: result[1]
       }))
-      .catch((err) => {
+      .catch(() => {
```

Comment:
Please handle this error by calling the error handler defined in `index.js`. That way you will be able to prevent `.catch` blocks from reducing test coverage and error handling will be centralized. You can do that by calling `catch(next)`